src/screens/ShowScreen.js

```diff
@@ -0,0 +1,18 @@
+import React, {useContext} from 'react'
+import {View, Text, StyleSheet} from 'react-native'
+import {Context} from '../context/BlogContext'
+
+const ShowScreen = ({navigation}) => {
+    const {state} = useContext(Context)
+const blogPost = state.find((blogPost) => blogPost .id === navigation.getParam('id'))
```

Comment:
use `find() `if the blog post id is equal to the navigation id we are looking for then show that blog